encled

```diff
@@ -172,6 +174,16 @@ def main(argv):
             print_status(d[0], d[1], d[3])
         return(0)
 
+    if argv[1].upper() == 'GOOD':
+        for d in devlist:
+            if ((d[3][1]).upper()).replace(" ", "") != 'FAULT_ON' and argv[2].lower() != 'off':
+                result = set_status(d[2], argv[2].lower())
```

Comment:
`argv[2].lower()` -> `'off'`, it's the same, but easier to read.
(in both 'if' and 'elif' sections).